service/db/dgclient.go

```diff
@@ -93,7 +93,7 @@ func NewDGClient(dgraphHost string) *DGClient {
 func (s DGClient) GetEntity(meta string, uuid string) (map[string]interface{}, error) {
 	q := `
 		{
-			objects(func: uid(` + uuid + `)) {
+			objects(func: uid(` + uuid + `)) @filter(eq(objtype,` + meta + `)) {
```

Comment:
should these queries use parameterized input?

essentially, could string concatenation like this potentially lead to SQL injection style attacks?